backend/routes/projectById.js

```diff
@@ -0,0 +1,34 @@
+const { parse } = require("url");
+
+const { missingProperty, invalidProperty } = require('../src/utils/errors');
+const stringify = require('../src/utils/stringify');
+const getAllProjects = require('../src/components/projects/getAllProjects');
+
+module.exports = async (req, res) => {
+  const { query: { project: projectParam } } = parse(req.url, true);
+
+  if (!projectParam) {
+    res.statusCode = 422;
+    res.end(stringify({
+      code: 422,
+      errors: [
+        missingProperty("project", "/get/project")
+      ]
+    }));
+  }
+
+  const allProjects = await getAllProjects();
+  const project = allProjects.find(({ id }) => id === projectParam);
+
+  if (!project) {
```

Comment:
Again, this maybe is more obvious because the value comes from 'find' function but still if the code above changed and 'project' would be of type 'number' for example. Just check for 'undefined'.

Reply:
I think we can safely assume that it won't return a number. It will always return a project object or undefined